### ISSUE TITLE
10.2.0 Compile issues with external package

### ISSUE THREAD

**Issue by ealmiladi** [vercel/next.js#24658]
### What version of Next.js are you using?

10.2.0

### What version of Node.js are you using?

14

### What browser are you using?

Chrome

### What operating system are you using?

macOK

### How are you deploying your application?

Docker

### Describe the Bug

```
Error: Failed to load /build
    at loadComponents (/node_modules/next/dist/next-server/server/load-components.js:1:1554)
```
After upgrading from 10.1.3 -> 10.2.0, my `/build` page just will not compile.

I've narrowed it down to a package that I'm using called `use-animate-presence`; including this package in my page file will break the page. However, if Webpack 4 was able to compile the package's contents correctly on Next 10.1.3, shouldn't Webpack 5? 

### Expected Behavior

Upgrading Next versions should be seamless and not require code level changes

### To Reproduce

To reproduce, start a blank NextJS project on 10.2.0, pull in the following package

```
import { useAnimatePresence } from 'use-animate-presence';
```

Notice your pages will not compile.

**Comment by ealmiladi** [comment]
Issue is resolved in `10.2.1-canary.3`. Closing this issue. Thanks!

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
